package.json

```diff
@@ -84,6 +84,7 @@
     "react-cookie": "^4.1.1",
     "react-dom": "^18.2.0",
     "react-stately": "^3.23.0",
+    "requests": "^0.3.0",
```

Comment:
is this package related?

Reply:
Sorry, as @pdehaan points out, `request` is related, corrected.